modyn/config/schema/pipeline/evaluation/config.py

```diff
@@ -46,15 +54,22 @@ class EvaluationConfig(ModynBaseModel):
         ),
         min_length=1,
     )
-    datasets: List[EvalDataConfig] = Field(
-        description="An array of all datasets on which the model is evaluated.",
+    datasets: dict[str, EvalDataConfig] = Field(
+        description="An array of all datasets on which the model is evaluated keyed by an arbitrary reference tag.",
```

Comment:
I guess I don't fully get that yet:

````
    arxiv_test:
      dataset_id: arxiv_test
```

Is there any ways to avoid this redundant specification of arxiv_test? Can't we use the reference tag as the name?